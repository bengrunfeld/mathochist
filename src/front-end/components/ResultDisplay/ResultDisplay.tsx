import React from 'react'

const ResultDisplay = () => (
  <div>
    <h1>ResultDisplay</h1>
  </div>
)

export default ResultDisplay
