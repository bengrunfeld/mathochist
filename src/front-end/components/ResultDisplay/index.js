import ResultDisplay from './ResultDisplay'

export default ResultDisplay
