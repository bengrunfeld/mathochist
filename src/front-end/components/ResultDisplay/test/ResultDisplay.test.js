import React from 'react'
import Enzyme, { shallow, mount, render } from 'enzyme'
import { MemoryRouter } from 'react-router-dom'

import ResultDisplay from '../ResultDisplay'

describe('<ResultDisplay />', () => {
  test('matches the snapshot', () => {
    const tree = mount(<ResultDisplay />)
    expect(tree).toMatchSnapshot()
  })
})
