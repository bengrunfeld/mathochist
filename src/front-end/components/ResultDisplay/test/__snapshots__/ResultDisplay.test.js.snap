// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultDisplay /> matches the snapshot 1`] = `
<ResultDisplay>
  <div>
    <h1>
      ResultDisplay
    </h1>
  </div>
</ResultDisplay>
`;
